index.rst

```diff
@@ -9,12 +9,31 @@
 Introduction
 ============
 
-This document describes the decisions made in the course of implementing the Data Backbone design from :dmtn:`122`.
+This document describes the decisions made in the course of implementing the Data Backbone design. 
+:dmtn:`122` provides an obsolete view of the starting point.
+
+The Data Backbone (DBB) concept is to provide data management (storage, tracking, and replication) for Rubin data products that reside in non-computational storage tiers. 
+It provides policy-defined operations for intra-site and inter-site data distribution, access latency requirements dependent upon the lifecycle of the data, data protection and recovery, data retention and eviction given cadences and predicted and observed on-demand usage, and efficient bulk recall of data which are spatially, temporally, or otherwise related in support of processing or export.
+
+This require services for:
+  - Replication and Transport
+  - Location and Metadata
```

Comment:
```suggestion
  - Replication and Transport, including the moving of data to archival storage
  - Location and Metadata, including the ingestion of data replicas into local Butler repositories
  
A related service is needed for transporting and tracking embargoed data, but since that is security-sensitive, it does not use the mechanisms described here.
```